### ISSUE TITLE
Cannot install on Windows

### ISSUE THREAD

**Issue by Epikem** [interpreters/pypreprocessor#10]
I installed pandoc and pypandoc and when I typed "pip install pypreprocessor", error occurs.

1. pip install pandoc
2. pip install pypandoc
3. pip install pypreprocessor

or 

1. install pandoc with msi
2. pip install pypandoc
3. pip install pypreprocessor

But both failed.

**Comment by evanplaice** [comment]
My mistake, you shouldn't need pandoc or pypandoc. Those are only used to generate the pypi documentation. Give me a few and I'll fix it.

**Comment by evanplaice** [comment]
OK, fixed. Released as 0.72

**Comment by Epikem** [comment]
Still can't install. Output is below.

```
PS xx> pip install pypreprocessor
Collecting pypreprocessor
  Using cached pypreprocessor-0.72.tar.gz
    Complete output from command python setup.py egg_info:
    Pypandoc not installed, skipping README.rxt creation
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "xx\pip-build-vjy1r10x\pypreprocessor\setup.py", line 31, in <module>
        license = open('LICENSE').read(),
    FileNotFoundError: [Errno 2] No such file or directory: 'LICENSE'

```
